### ISSUE TITLE
Drop "siglum" on "Sequence" model?

### ISSUE THREAD

**Issue by ahankinson** [DDMAL/CantusDB#1534]
The "siglum" text field seems like it's a leftover from the migration. Since the Sequence model is already connected to the source, and can get the source information there, it's likely this will gradually lead to divergences between what is in the text field, and what source the sequence is actually connected to.

Unless I'm missing something, the siglum field can probably be dropped? 

Maybe this is one that @annamorphism can weigh in on?

**Comment by annamorphism** [comment]
I agree, in the long-term. Shorter term, there are some existing divergences in the Bower database that need to be cleaned up--cases where sequences were not correctly attached to a source in the first place when the data was put in. These are all attached to the "Unknown Source" right now (https://cantusdatabase.org/source/1000004) but many of the sequences do have siglum information that will help them get attached to an actual source at some point. 

So: I say this could be made something that most people don't see, and nobody edits or add to, but that is still accessible somewhere for people who need it to clean up the Bower data.

**Comment by annamorphism** [comment]
seems like this was fixed by an earlier PR? If it comes up again in cleaning the Bower sources we can revisit. closing.
